scripts/Makefile

```diff
@@ -0,0 +1,60 @@
+define HELPBODY
+Available commands:
+
+	make help                             - this thing
+
+	make load_transfer                    - load send token API
+	make load_get_balance                 - load read token API
+	make load_batch_status                - load batch satus API
+	make load_user_pub_keys               - load read user pub keys API
+	make load_certificate_creation        - load certificate creation
+	make load_certificate_check           - load certificate check if cert is valid
+	make load_certificate_delete          - load certificate deletion
+	make load_personal_create          	  - load personal public key creation
+	make load_personal_delete          	  - load personal public key deletion
+	make load_personal_keys          	  - load personal read of public keys
+
+endef
+
+export HELPBODY
+
+REST_API_URL=http://localhost:8080
+
+CLIENTS=1
+THREADS=1
+TIME=30
+
+ts := $(shell /bin/date "+%Y%m%d%H%M%S")
+
+help:
+	@echo "$$HELPBODY"
+
+load_transfer:
+	wrk -c ${CLIENTS} -t ${THREADS} -d ${TIME} --latency -s ./token_post.lua ${REST_API_URL}/api/v1/token
```

Comment:
A little note on this test: as Sawtooth framework can execute independent transactions in parallel this test runs on the worst case (all transactions are dependent).

Reply:
@eugene-babichenko are we testing sawtooth or our node? In current implementation we have only one transaction per batch, maybe need to reimplement the functionally and after get back for testing this? Or I miss understood the problem.